src/explore-education-statistics-admin/src/pages/methodology/createMethodologyPage.tsx

```diff
@@ -0,0 +1,223 @@
+import SummaryList from '@common/components/SummaryList';
+import SummaryListItem from '@common/components/SummaryListItem';
+import Yup from '@common/lib/validation/yup';
+import { FormikProps } from 'formik';
+import orderBy from 'lodash/orderBy';
+import React, { useEffect, useState } from 'react';
+import { RouteComponentProps } from 'react-router';
+import { CreateMethodologyRequest } from '@admin/services/methodology/types';
+import {
+  DayMonthYearInputs,
+  dateToDayMonthYear,
+  dayMonthYearValuesToInputs,
+  dayMonthYearInputsToDate,
+} from '@common/services/publicationService';
+import { validateMandatoryDayMonthYearField } from '@admin/validation/validation';
+
+interface MatchProps {
+  topicId: string;
+}
+
+interface FormValues {
+  methodologyTitle: string;
+  selectedContactId: string;
+  scheduledPublishDate: DayMonthYearInputs;
+}
+
+const serverSideValidationHandler = handleServerSideValidation(
+  errorCodeToFieldError(
+    'SLUG_NOT_UNIQUE',
+    'methodologyTitle',
+    'Choose a unique title',
+  ),
+);
+
+interface CreateMethodologyModel {
+  contacts: ContactDetails[];
+}
+
+const CreateMethodologyPage = ({
+  match,
+  history,
+}: RouteComponentProps<MatchProps>) => {
+  const [model, setModel] = useState<CreateMethodologyModel>();
+
+  useEffect(() => {
+    Promise.all([service.getPublicationAndReleaseContacts()]).then(
+      ([contacts]) => {
+        setModel({
+          contacts,
+        });
+      },
+    );
+  }, []);
+
+  const submitFormHandler = async (values: FormValues) => {
+    const submission: CreateMethodologyRequest = {
+      title: values.methodologyTitle,
+      publishScheduled: dayMonthYearInputsToDate(values.scheduledPublishDate),
+      contactId: values.selectedContactId,
+    };
+
+    const createdMethodology = await methodologyService.createMethodology(
+      submission,
+    );
+
+    // TODO: redirect to the methodology summary
+    history.push(`/methodology/${createdMethodology.id}`);
+  };
+
+  const cancelHandler = () => history.push('/methodology');
+
+  const getSelectedContact = (
+    contactId: string,
+    availableContacts: ContactDetails[],
+  ) =>
+    availableContacts.find(contact => contact.id === contactId) ||
+    availableContacts[0];
+
+  const formId = 'createMethodologyForm';
+
+  return (
+    <Page
+      wide
+      breadcrumbs={[
+        {
+          name: 'Manage methodology',
+          link: '/methodology',
+        },
+        {
+          name: 'Create new methodology',
+        },
+      ]}
+    >
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <h1 className="govuk-heading-xl">Create new methodology</h1>
+        </div>
+        <div className="govuk-grid-column-one-third">
+          <RelatedInformation heading="Help and guidance">
+            <ul className="govuk-list">
+              <li>
+                <Link to="/documentation" target="blank">
```

Comment:
Does this need a specific URL?

Reply:
And does "blank" need to be "_blank"?

Reply:
Specific URL does not currently exist so pointing to general documentation for now